macros/get_call_columns.sql

```diff
@@ -26,17 +26,17 @@
     {"name": "updated_at", "datatype": dbt.type_timestamp()}
 ] %}
 
-{% if target.type == 'snowflake' %}
-    {{ columns.append({"name": "FROM", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }}
-{% else %}
-    {{ columns.append({"name": "from", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }}
-{% endif %},
-
-{% if target.type == 'snowflake' %}
-    {{ columns.append({"name": "TO", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }}
-{% else %}
-    {{ columns.append({"name": "to", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }}
-{% endif %},
+{% if target.type == 'snowflake' %} 
+    {{ columns.append({"name": "FROM", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }} 
+{% else %} 
+    {{ columns.append({"name": "from", "datatype": dbt.type_string(), "quote": True, "alias": "call_from"}) }} 
+{% endif %}
+ 
+{% if target.type == 'snowflake' %} 
+    {{ columns.append({"name": "TO", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }} 
+{% else %} 
+    {{ columns.append({"name": "to", "datatype": dbt.type_string(), "quote": True, "alias": "call_to"}) }} 
+{% endif %}
```

Comment:
The trailing comma could potentially cause issue. We should remove the comma from the last item in the macro.

Reply:
removed